query/src/main/scala/filodb/query/exec/rangefn/RateFunctions.scala

```diff
@@ -127,6 +180,51 @@ object DeltaFunction extends RangeFunction {
   }
 }
 
+object PeriodicIncreaseFunction extends RangeFunction {
```

Comment:
Increase should be an alias for `sum_over_time` for delta counters and histograms.

Reply:
👍 update the implementation to use `sumovertime`